app/qml/Status/Application/MainView/MainView.qml

```diff
@@ -2,34 +2,61 @@ import QtQuick
 import QtQuick.Layouts
 import QtQuick.Controls
 
+import Status.Application
```

Comment:
I like how clean this file looks and how it is structured as a global with the specific navigation bar per section, a global banner for the global app and the main loader.